tests/integration/inventories/vyos.yaml

```diff
@@ -15,5 +15,5 @@ appliance:
     # NOTE(pabelanger): With stable-2.9 / stable-2.11 we need to force
     # /bin/python3.8 and not discover /usr/libexec/platform-python (python3.6)
     # ansible_python_interpreter: /bin/python3.8
-    collection_name: vyos
-    collection_namespace: vyos
+    collection_name: "{{ collection_name }}"
+    collection_namespace: "{{ collection_namespace }}"
```

Comment:
we don't want to change this file, we should update the playbook I linked before. Where we overwrite this file with the zuul job.vars if defined.